manifests/opsmanager.pp

```diff
@@ -0,0 +1,59 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String[1] $user                                = $mongodb::params::opsmanager_user,
+  String[1] $group                               = $mongodb::params::opsmanager_group,
+  Variant[Boolean, String[1]] $ensure            = $mongodb::params::opsmanager_ensure,
+  String[1] $package_name                        = $mongodb::params::opsmanager_package_name,
+  Boolean $package_ensure                        = $mongodb::params::opsmanager_package_ensure,
+  Boolean $service_enable                        = $mongodb::params::opsmanager_service_enable,
+  Boolean $service_manage                        = $mongodb::params::opsmanager_service_manage,
+  String[1] $service_name                        = $mongodb::params::opsmanager_service_name,
+  String[1] $download_url                        = $mongodb::params::opsmanager_download_url,
+  String[1] $mongo_uri                           = $mongodb::params::opsmanager_mongo_uri,
+  String[1] $hostname                            = $mongodb::params::opsmanager_hostname,
+  String[1] $protocol                            = $mongodb::params::opsmanager_protocol,
+  Stdlib::Port $port                             = $mongodb::params::opsmanager_port,
+  String[1] $opsmanager_url                      = $mongodb::params::opsmanager_url,
+  String[1] $client_certificate_mode             = 'None',
+  String[1] $from_email_addr                     = 'from@yourdomain.com',
+  String[1] $reply_to_email_addr                 = 'replyto@yourdomain.com',
+  String[1] $admin_email_addr                    = 'admin@yourdomain.com',
+  String[1] $email_dao_class                     = 'com.xgen.svc.core.dao.email.JavaEmailDao', #AWS SES: com.xgen.svc.core.dao.email.AwsEmailDao or SMTP: com.xgen.svc.core.dao.email.JavaEmailDao
+  String[1] $mail_transport                      = 'smtp', #smtp or smtps
```

Comment:
If it's only `smtp` or `smtps`, should it be an `Enum['smtp', 'smtps']` instead?